samples/aspnetcore/security/README.md

```diff
@@ -1,3 +1,3 @@
 # Security samples
 
-Please find additional security and authentication related samples here: https://github.com/aspnet/AspNetCore/tree/master/src/AuthSamples/samples
+Please find additional security and authentication related samples here: https://github.com/aspnet/AspNetCore/tree/main/src/AuthSamples/samples
```

Comment:
Actually this is the wrong url

Reply:
https://github.com/dotnet/aspnetcore/tree/main/src/Security/samples

Reply:
Thanks, fixed